gossip/src/crds_value.rs

```diff
@@ -95,30 +95,31 @@ impl CrdsValueLabel {
 }
 
 impl CrdsValue {
-    pub fn new_unsigned(data: CrdsData) -> Self {
+    pub fn new(data: CrdsData, keypair: &Keypair) -> Self {
+        let bincode_serialized_data = bincode::serialize(&data).unwrap();
```

Comment:
need an `.expect("failed to serialize CrdsData")` here instead of an `unwrap()`?

Reply:
I feel like that just adds some verbosity without any additional context to debug that in this case.

Reply:
ok ya fair enough